roles/wazuh/ansible-wazuh-manager/vars/aws_creds.yml

```diff
@@ -0,0 +1,11 @@
+#---
+#monitor_aws:
```

Comment:
This default vars are duplicated, they are already present in ` roles/wazuh/ansible-wazuh-manager/defaults/main.yml` right?

I agree that our manager and agent defaults is really extensive but I think it's better to refactor them all at once instead of creating external vars little by little.

Reply:
This was more for vaulting credentials